helm/grafana/Chart.yaml

```diff
@@ -1,14 +1,16 @@
 apiVersion: v2
 name: grafana
-version: 0.4.0
-appVersion: 8.4.2
```

Comment:
That's a major bump. Any breaking changes?

I would like to see changelog somewhere or at least important updates and a major version bump for `version`

Reply:
No breaking changes no as Grafana mostly keeps compatibility hence I did not see the need to bump the major version but I can do it

Reply:
https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0/#breaking-changes

Reply:
The things changed by us are merely disabling pspUseAppArmor and enabling psp